zuul.d/projects.yaml

```diff
@@ -1,4 +1,16 @@
 ---
+
 - project:
     name: ansible/ansible
     default-branch: devel
+
+- project:
+    name: ansible/awx
+    default-branch: devel
+    templates:
```

Comment:
this is not valid, and can be removed

Reply:
@pabelanger why is it invalid? We have such definitions in `dci-config` and its been working fine.

Reply:
Ah, sorry. Lack of coffee, my brain read this as Job stanza, not project. Sorry for the noise.